### ISSUE TITLE
Updating the wiki for EteSync 2.0

### ISSUE THREAD

**Issue by Salt-Factory** [etesync/server#54]
Hi tasn,

Today I finally found the time to update my self hosted EteSync server to the EteBase server.

For the EteSync 1.0 installation, I created 4 new wiki pages detailing how to do this.
I followed these for EteBase, and there are no major differences except the name change, and that you should checkout the correct branch when cloning.

My question now is: do I change these wiki pages to represent the move to EteSync 2.0?
Or would you prefer that I create 4 separate pages, specifically for EteBase? Then in the future, we could simply remove the old EteSync 1.0 installation.

**Comment by tasn** [comment]
How was the setup? Was it easy? Hard? Different? Have you noticed that there's now also a media dir so that you need to make sure to keep that backed up? Would love to hear your feedback!

Maybe create the 4 separate pages, and add links to the new pages from the old pages including maybe a link to the 2.0 blog just so people know what's up and that if they are new they should probably use v2? What do you think?

**Comment by Salt-Factory** [comment]
The setup was about the same as setup for v1.0!
I'm not 100% sure yet if the setup that I followed was correct however, seeing as I have not found anything about the media dir, and that I am still stuck on #55.

For the wiki, I think creating a separate set of pages would then be a good idea.
Also, I spent some time browsing the past issues, and in #26, @daftaupe suggests to use Gunicorn instead of uWSGI as WSGI, as it is easier to set up. If I find time, I'll try to convert my setup to use Gunicorn and I'll change the wiki accordingly.
